dev.linwood.butterfly.json

```
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.linwood.butterfly",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butterfly",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "butterfly",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -r FlutterApp /app/",
+                "chmod +x /app/FlutterApp/butterfly",
+                "mkdir -p /app/bin",
+                "ln -s /app/FlutterApp/butterfly /app/bin/butterfly",
+                "install -Dm644 dev.linwood.butterfly.appdata.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml",
+                "install -Dm644 dev.linwood.butterfly.svg -t /app/share/icons/hicolor/scalable/apps/dev.linwood.butterfly.svg",
```

Should be
```
"install -Dm644 dev.linwood.butterfly.svg -t /app/share/icons/hicolor/scalable/apps/"
```

---

sorry that's my fault
```suggestion
                "install -Dm644 dev.linwood.butterfly.svg -t /app/share/icons/hicolor/scalable/apps/",
```